### PR TITLE
Changes for rds app upgrade and adding of new monitors for AWSO 2.8.0

### PR DESCRIPTION
**Pull request by sumoanema**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [ ] Yes, I've run `yarn install`
- [X] No, does not apply to me

## Purpose of this pull request

This pull request is for introducing changes in AWSO docs for - rds app upgrade and adding of new monitors for AWSO 2.8.0

[](https://sumologic.atlassian.net/browse/SUMO-236917)

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [X] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

**Review by JV0812 — APPROVED** [review]
LGTM

File: docs/observability/aws/integrations/amazon-rds.md [hunk 1]
```diff
@@ -149,6 +150,18 @@ The Amazon RDS app uses the following logs and metrics:
    "logGroup":"/aws/rds/instance/rds-dbinstance-1/audit"
 }
 ```
+
+```json title ="Authentication Failure (PostgreSQL Log)"
+{"timestamp":1705859815000,
+"message":"2024-01-21 17:56:55 UTC:3.92.54.14(57164):postgresql@postgresql:[43627]:FATAL:  password authentication failed for user \"postgresql\""
```
Comment (kimsauce) [inline docs/observability/aws/integrations/amazon-rds.md:156]: This line ends with two double-quotes. Is that intentional?

**Review by kimsauce — APPROVED** [review]
I piggybacked on this PR and fixed something unrelated, but needed: cleaned up HTML character syntax.

File: docs/observability/aws/integrations/amazon-rds.md (hunk 1 above)
Comment (JV0812) [reply docs/observability/aws/integrations/amazon-rds.md:156]: @kimsauce yes Kim. You can see a  double-quotes opening at the start of value.

**Comment by JV0812** [comment]
This PR is consolidated with the PR: https://github.com/SumoLogic/sumologic-documentation/pull/3892. So closing this PR.

**Pull request closed without merging**